styleguide/task.py

```diff
@@ -21,6 +21,7 @@
 # List of regexes for folders which contain generated files
 genFolderExclude = \
     [name + sep for name in [
+     ".vscode",
```

Comment:
The "." should be escaped with a "\" for regexes. Otherwise, "." matches any character.